src/jni_core/src/jvm_runtime.rs

```diff
@@ -48,10 +48,8 @@ impl JavaVmWrapper {
         let libs_path = if let Ok(libs_path) = std::env::var("CONNECTOR_LIBS_PATH") {
             libs_path
         } else {
-            return Err(ErrorCode::InternalError(
-                "environment variable CONNECTOR_LIBS_PATH is not specified".to_string(),
-            )
-            .into());
+            tracing::warn!("environment variable CONNECTOR_LIBS_PATH is not specified, so use default path `./libs` instead");
```

Comment:
It's better to join the env.args[0](which is the path of the binary) and `./libs` to get the path. The current way only works correctly when pwd is the same as the directory of `risingwave` binary (where we can run `./risingwave`). If users run as `/.../.../risingwave`, we may not get the correct libs path.

Reply:
https://doc.rust-lang.org/std/env/fn.current_exe.html may work.